### ISSUE TITLE
Recent update breaks GraphQL

### ISSUE THREAD

**Issue by steveDL** [verbb/navigation#239]
**Description**

Upgrade to 1.4.10 and you will get `GraphQL error: Cannot query field "nodes" on type "Query".`

Downgrading has fixed the issue.

**Steps to reproduce**
1. Upgrade to 1.4.10 
2. try and query nodes

**Additional info**
- Plugin version:1.4.10
- Craft version: Craft CMS 3.5.17.1
- Multi-site: No
- Propagate nodes: Yes or No

**Comment by engram-design** [comment]
Can't seem to replicate this with a simple query using the GraphQL Explorer:

```
{
    nodes (navHandle: "mainMenu") {
        url
    }
}
```

Can you advise what query you're using, and from where you're calling it from, if not from the GraphQL Explorer?

**Comment by engram-design** [comment]
Any update on this?

**Comment by engram-design** [comment]
I should mention, this could _possibly_ be tied up to the schema permissions introduced in [1.4.6](https://github.com/verbb/navigation/releases/tag/1.4.6). Can you let me know if you're using your own custom schema or just using the public schema.

If the former, you'll need to enable permissions to query nodes on your schema.

**Comment by magicspon** [comment]
multiple queries kills it

```graphql
query MyQuery {
  sitenav: nodes(navHandle: "sitenav", level: 1) {
    title
  }
  subnav: nodes(navHandle: "subnav", level: 1) {
    title
  }
}
```

**Comment by engram-design** [comment]
@magicspon Pretty sure this might be related to Craft 3.6+ as again, I can't replicate that issue on Craft 3.5.x or 3.6.x

**Comment by engram-design** [comment]
I can't seem to replicate this behaviour no matter what @magicspon. But I believe the original issue for @steveDL would be permissions based, in that you need to opt-in to querying navigations, if you're using schemas.

**Comment by daltonrooney** [comment]
@engram-design I just upgraded to Craft 3.6.11.2 and Navigation 1.4.15 and I think I'm experiencing this too. I'm using custom schemas and I've enabled `View all navigations` for my schema. When I try to query a node, I get an error like:

```
 "debugMessage": "Tried to load an unregistered type \"primary_Node\". This can indicate both a typo in the query or an issue with the schema used.",
      "message": "Internal server error",
      "extensions": {
        "category": "internal"
      },
```

When I explicitly add my navigations to the schema, then my queries work as expected. 

Edit: just for reference, adding `View all forms` to my schema does work as expected for Formie.

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/navigation` requirement in `composer.json` to:

```
"require": {
  "verbb/navigation": "dev-craft-3 as 1.4.15",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.16](https://github.com/verbb/navigation/releases/tag/1.4.16)
